src/details/ListingSearch.js

```diff
@@ -294,16 +243,20 @@ const ListingsSearch = () => {
                                         </select>
                                     </div>
                                 </div>
-                                <div className="row p-4" style={{backgroundColor: "black"}}>
-                                    <div className="col-md-12 p-5" style={{backgroundColor: "white"}}>
+                                <div className="row p-4 listing-area">
+                                    <div className="col-md-12 p-5 listing-content">
                                         <div className="row">
                                             <p className={"mr-auto"}>100K WINNING STUD</p>
                                             <p>$5000</p>
                                         </div>
                                         <div className="row">
                                             <div className="col-md-4">
                                                 <img className="" src={horse1} alt={"img"}/>
-                                                <AiOutlineHeart/>
+                                                {liked ? <AiFillHeart onClick={() => {
+                                                    setLiked(!liked)
+                                                }}/> : <AiOutlineHeart onClick={() => {
+                                                    setLiked(!liked)
+                                                }}/>}
```

Comment:
Right now this is hardcoded as only 1 listing.  This will be filled with `listingData`.  It will need a `Map` function.

It should be something like below
```JS
                            {listingData.map((listing) => (
                                <img className={"w-100"} src={listing.mainImage} />
                               <p>{listing.description}</p>
                            ))}
```